### ISSUE TITLE
Publicize memberships in canada-ca GitHub org

### ISSUE THREAD

**Issue by patcon** [canada-ca/welcome#22]
I noticed that all the folks who participate in this GitHub org are not publicizing their membership on the member page:
https://github.com/orgs/canada-ca/people

Related, I've been struck by how active non-male voices are in this particular issue queue, which is refreshing and inspiring given the normal culture of GitHub.

I realize that the choice not to publicize might be intentional, but I wanted to surface it for discussion. Would it be possible/desirable to encourage folks to publicize their membership in the GitHub org?
https://help.github.com/articles/publicizing-or-hiding-organization-membership/

cc: @MaryBethBaker @laurawesley

**Comment by patcon** [comment]
Alternatively, this might also be a helpful way to communicate who is participating in a repository and associated conversations :)
https://github.com/kentcdodds/all-contributors

**Comment by MaryBethBaker** [comment]
No private repos - only open for us!

//cc @laurawesley

**Comment by laurawesley** [comment]
mmm you are right @patcon - all the members are set to private. I think this is a default thing; not an intentional thing. Am clicking around...

**Comment by patcon** [comment]
totally -- yeah, wasn't implying anything about all your openness! (Because you are all wonderful at this.)

Usually in open source project onboarding, we add an extra little step to walk people through publicizing, so it's a "known problem" that often needs some attention ;)
https://trello.com/c/yRMYHQie/13-get-memberized-on-our-github-teams

Anyhow, just wanted to raise it, but perhaps no action item here, and I can close.

Know if there's a GitHub for civil servants or any such resource that we could add this to? Any go-to resources that you use to lure people into this github org? ;)

**Comment by nschonni** [comment]
It is listed on https://government.github.com/, but Canada doesn't have an equivalent to https://usdigitalregistry.digitalgov.gov/ AFAIK

**Comment by chrismajewski** [comment]
@patcon You're right, and I should be more active.  Here's me calling me out on that.

I'm on the public list.

**Comment by MaryBethBaker** [comment]
I'm going to contact members and see if they want to publish their affiliation. If that's cool with them they can go ahead. Is this enough for me to close this issue? @patcon @laurawesley @nschonni @chrismajewski

**Comment by patcon** [comment]
But maybe also we could create an "onboarding" that has this consideration as a step for when new members are added? This doc could also be a place to baseline aspiring culture of this github org -- not sure if there are other things you wished to point out to new members :)

**Comment by patcon** [comment]
I could see that being a good action item to not only address this now, but going forward. That would seem to be a good way to close out as addressed :tada:

**Comment by MaryBethBaker** [comment]
Hi Pat,

There’s another issue linked to a markup document that will contain a management plan. Onboarding could go there... although I’m more likely to point colleagues to already created and maintained documentation such as WET documentation. Given the number of people joining, maintaining, and at present limited time to dedicate to documentation. 

I’m going to keep these two requests separate.

Mary Beth
